.nvmrc

```diff
@@ -1 +1 @@
-v22.17.0
+v22.20.0
```

Comment:
There's a case to be made for us to simply do `v22` here, but that's likely for a separate PR. Or have dependabot bump that I guess?